SpatialGDK/Source/SpatialGDK/Private/EngineClasses/RPCContainer.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/RPCPayload.h"
+#include "SpatialConstants.h"
+
+namespace SpatialGDK
```

Comment:
I would ask @improbable-valentyn but I think our stance on namespaces is that if the class/struct doesn't have some kind of Unreal prefix like T or F, we would then put it into the namespace. For example, UnrealObjectRef.h is not in a namespace since it's called FUnrealObjectRef. So I believe this shouldn't be in a namespace either.

Reply:
According to naming convention, I believe I should add F prefix -> FRPCContainer. Should I still remove namespace SpatialGDK?

Reply:
Yes, you can remove it from the namespace.